### PR TITLE
Add the job publication widget

### PR DESCRIPTION
**Pull request by 99999989**
screenshot 2019-01-09 at 12 01 03

### CODE REVIEW

File: alv-portal-ui/src/app/widgets/job-publication-widget/job-publication-widget.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="alv-panel pb-4">
+  <h2 class="panel-title">
+    <i class="panel-icon fa fa-search mr-2"
```
Comment (birom) [inline alv-portal-ui/src/app/widgets/job-publication-widget/job-publication-widget.component.html]: I think we need here ``fa-edit``.

File: alv-portal-ui/src/app/dashboard/company-dashboard/company-dashboard.component.html [hunk 2]
```diff
@@ -1,7 +1,10 @@
+<alv-job-publication-widget class="d-block mt-4">
```
Comment (birom) [inline alv-portal-ui/src/app/dashboard/company-dashboard/company-dashboard.component.html]: We also need to add the widget to the ``company-home.component.html`` and ``pav-home.component.html`` pages.

File: alv-portal-ui/src/app/widgets/job-publication-widget/job-publication-widget.component.html (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/widgets/job-publication-widget/job-publication-widget.component.html]: I've changed it to `fa-edit`

File: alv-portal-ui/src/app/dashboard/company-dashboard/company-dashboard.component.html (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/dashboard/company-dashboard/company-dashboard.component.html]: I've added them  according to https://rlwa8z.axshare.com/#g=1&p=arbeit_swiss_services_-_nicht_eingeloggt_red

**Pull request merged**